tests/testsuite/cargo_add/git_multiple_packages_features/mod.rs

```diff
@@ -0,0 +1,43 @@
+use cargo_test_support::compare::assert_ui;
+use cargo_test_support::prelude::*;
+use cargo_test_support::Project;
+
+use cargo_test_support::curr_dir;
+
+#[cargo_test]
+fn case() {
+    cargo_test_support::registry::init();
+
+    let mut members_manifest = cargo_test_support::basic_manifest("packmain", "0.1.1+mainpack");
+    members_manifest.push_str("members = [pack1, pack2]");
+
+    let mut features_manifest = cargo_test_support::basic_manifest("pack2", "0.1.1+pack2");
+    features_manifest.push_str("[features]\nfeat-pack2 = []");
+
+    let project = Project::from_template(curr_dir!().join("in"));
+    let project_root = project.root();
+    let cwd = &project_root;
+    let git_dep = cargo_test_support::git::new("git-package", |project| {
+        project
+            .file("Cargo.toml", &members_manifest)
+            .file(
+                "pack1/Cargo.toml",
+                &cargo_test_support::basic_manifest("pack1", "0.1.1+pack1"),
+            )
+            .file("pack1/src/lib.rs", "")
+            .file("pack2/Cargo.toml", &features_manifest)
+            .file("pack2/src/lib.rs", "")
```

Comment:
nit: Could you give these package names a more descriptive name so its easier to inspect the test and tell what is intended